src/dc_job.c

```diff
@@ -366,6 +320,12 @@ static void dc_job_do_DC_JOB_SEND_MSG_TO_SMTP(dc_context_t* context, dc_job_t* j
 			goto cleanup; /* unrecoverable */
 		}
 
+		/* add SELF to the recipient list (it's no longer used elsewhere, so a copy of the whole list is needless) */
+		if (clist_search_string_nocase(mimefactory.recipients_addr, mimefactory.from_addr)==0) {
+			clist_append(mimefactory.recipients_names, NULL);
+			clist_append(mimefactory.recipients_addr,  (void*)dc_strdup(mimefactory.from_addr));
+		}
+
```

Comment:
this is effectively a BCC, i.e. recipients are not added to the "To" or "CC" headers, right?

Reply:
exactly :)